fizz-buzz-streams/src/test/java/com/pubsap/eng/streams/FizzBuzzAggregatorTest.java

```diff
@@ -0,0 +1,103 @@
+package com.pubsap.eng.streams;
+
+import com.pubsap.eng.schema.*;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FizzBuzzAggregatorTest {
+
+    @Test
+    public void aggregatorShouldUpdateFizz() {
+        //Given
+        InputKey givenInputKey = new InputKey("client-1");
+        Item givenItem = new Item(ItemValue.Fizz);
+        Output givenInitOutput = new Output(1,0,0);
+
+        //when
+        Output aggResult = FizzBuzzAggregator.aggregator.apply(givenInputKey, givenItem, givenInitOutput);
+
+        //Then
+        assertEquals(aggResult,new Output(2,0,0));
+    }
+
+    @Test
+    public void aggregatorShouldUpdateFizzFromInit() {
```

Comment:
⚠️  Most Important Here ⚠️ 

Don't test the apply method. Use the Topology test driver again. 

This way you will also verify that the buffering and serde mechanism is OK!